doc/classes/FontData.xml

```diff
@@ -600,5 +600,8 @@
 		<member name="style_name" type="String" setter="set_font_style_name" getter="get_font_style_name" default="&quot;&quot;">
 			Font style name.
 		</member>
+		<member name="subpixel_positioning" type="int" setter="set_subpixel_positioning" getter="get_subpixel_positioning" enum="TextServer.SubpixelPositioning" default="1">
+			Font glyph sub-pixel positioning mode.
```

Comment:
Could we document _why_ you would use this? Basically that it trades performance for sharper rendering, especially for smaller font sizes, and may not have much effect for large font sizes.

We could also recommend the AUTO setting and point to the project setting.